qiskit_experiments/curve_analysis/curve_data.py

```diff
@@ -134,3 +135,119 @@ class ParameterRepr:
 
     # Unit
     unit: Optional[str] = None
+
+
+# pylint: disable=invalid-name
+class FitOptions:
+    """Collection of fitting options.
+
+    This class is initialized with a list of parameter names,
+    and automatically format given initial parameters and boundaries based on it.
+
+    This class provides ``__hash__`` and ``__eq__`` methods to evaluate duplication.
+    """
+
+    def __init__(self, parameters: List[str]):
+        """Create a new fit options."""
```

Comment:
Even though this is internal I'd still add a docstring for the args.

Reply:
Good catch. Updated in https://github.com/Qiskit/qiskit-experiments/pull/390/commits/bcc34703d98e13bcb17848eed9742526046b9f31